ocaml/xapi/pool_db_backup.ml

```diff
@@ -250,7 +250,7 @@ let fetch_database_backup ~master_address ~pool_secret ~force =
 (* Master sync thread *)
 let pool_db_backup_thread () = Debug.with_thread_named "pool_db_backup_thread" (fun () ->
     Server_helpers.exec_with_new_task "Pool DB sync" (fun __context ->
-        while (true) do
+        while (false) do
```

Comment:
Should this be based on a configuration value in `/etc/xapi.conf`? If enabled we use the new mechanism (and have `false` here), if not we use the old mechanism. That way we can switch between the two without rebuilding.

Reply:
I've changed this to use the config file. But this will probably change again very soon re: Rob's comments